oppia/oppia
# Hide Playthrough Issues on Exploration Editor Stats Tab if Exploration is not Whitelisted

Playthroughs should only be visible and recorded when a particular exploration is whitelisted. This allows us to continue working on changes and avoid recording potentially unsafe data until we're confident that Playthroughs are ready for general usage.

**Testing Steps**:
1. Go to: Admin Page > Activities
    - Load Single Exploration 'Hola' (exploration id: '7')
2. Go to: Admin Page > Config
    - Add element '7' to "The set of exploration IDs for recording issues and playthroughs"
    - Set "The probability of recording playthroughs" to 1 (i.e. 100%)
    - Save changes
3. Play Hola exploration
    - Answer with 'Hola' and move to next card
    - Immediately navigate away from the exploration (and accept warning)
    - This should log an "Early Quit Playthrough"
4. Go to: Hola's Exploration Editor > Stats Tab
    - Make sure Playthrough directive is visible and has one issue
5. Go to: Admin Page > Config
    - Remove '7' from "The set of exploration IDs for recording issues and playthroughs"
    - Save changes
6. Go to: Hola's Exploration Editor > Stats Tab
    - Make sure Playthrough directive is *not* visible
7. Play Hola exploration
    - Answer with 'x' several times (~5s apart)
    - Navigate away from the exploration (and accept warning)
    - This should log a "Multiple Incorrect Answer Playthrough"
8. Go to: Admin Page > Config
    - Add element '7' to "The set of exploration IDs for recording issues and playthroughs"
9. Go to: Hola's Exploration Editor > Stats Tab
    - Make sure Playthrough directive is visible and still has exactly one issue
10. Play Hola exploration
    - Answer with 'y' several times (~5s apart)
    - Navigate away from the exploration (and accept warning)
    - This should log a "Multiple Incorrect Answer Playthrough"
11. Go to: Hola's Exploration Editor > Stats Tab
    - Make sure Playthrough directive is visible and has exactly two issues
